spec.html

```diff
@@ -13728,6 +13728,7 @@ <h1>Runtime Semantics: Evaluation</h1>
           1. Assert: Type(_newTarget_) is Object.
           1. Let _func_ be ? GetSuperConstructor().
```

Comment:
now that the check is removed, this can no longer throw
```suggestion
          1. Let _func_ be ! GetSuperConstructor().
```